cmd/removeEnumerator.go

```diff
@@ -150,29 +151,79 @@ func removeBfsResources(cca *cookedCopyCmdArgs) (successMessage string, err erro
 	// parse the given source URL into parts, which separates the filesystem name and directory/file path
 	urlParts := azbfs.NewBfsURLParts(*sourceURL)
 
-	// list of files is given, record the parent path
-	parentPath := urlParts.DirectoryOrFilePath
-	successCount := 0
-
 	if cca.listOfFilesChannel == nil {
-		return removeSingleBfsResource(urlParts, p, ctx, cca.recursive)
+		successMsg, err := removeSingleBfsResource(urlParts, p, ctx, cca.recursive)
+		if err != nil {
+			return err
+		}
+
+		glcm.Exit(func(format common.OutputFormat) string {
+			if format == common.EOutputFormat.Json() {
+				summary := common.ListJobSummaryResponse{
+					JobStatus:          common.EJobStatus.Completed(),
+					TotalTransfers:     1,
```

Comment:
Is it just the format that needs to be consistent with other types of removal?  NOT the count?  E.g. remove a folder with this and the count will be 1. Remove a folder with anything else, and the count will be recursive count of contents.

Reply:
Yes, we couldn't get the info on how many child items were deleted. They jus need the data to be easier to process.